README.md

```diff
@@ -0,0 +1,169 @@
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rails Budget App <a name="about-project"></a>
+
+**Budget app** An appliation help to track users expenses. The app has authunication for security reasons.
+
+<a href="https://www.loom.com/share/a1e6cae6108346f9a3e74d6c53418c0f?sid=3c9ddcfe-e7ab-4f19-9488-f871d2b4db1a">Presentation vedio about the project</a>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>localhost</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Keep track of all your recipes, ingredients, and inventory]**
+- **[Create recipes, and generate shopping lists]**
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- You'll need any code editor 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/getssh/budget_app
+  cd Rails-Recipe-App
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+  rails db:create
+  rails db:migrate
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rails db:migrate RAILS_ENV=test
+  rspec
+```
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Getayawkal Tamrat**
+
+- GitHub: [@getssh](https://github.com/getssh)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/getayawkal-tamrat/)
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Improve UI design]**
+- [ ] **[Improve the desktop version]**
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can give it a star!
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```

Comment:
- [ ] It's a nice and smart thing to give appropriate credit to **[Gregoire Vella on Behance](https://www.behance.net/gregoirevella)**, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.

Reply:
**Please note that this is part of the project requirement** Kindly make sure appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella), the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).